### DISCUSSION TITLE
Blade::render is not working?

### DISCUSSION THREAD

**Discussion by virla01** [filamentphp/filament#14379] [Help]
### Package

Other

### Package Version

v3.2

### How can we help you?

Could it be that Blade::render is not working properly and not escaping the quotes so that instead of seeing the code it comes out as html?
I hope it helps, thanks!

**Comment by leandrocfe** [comment]
What code are you using?

**Comment by virla01** [comment]
https://stackoverflow.com/questions/78412353/how-to-add-an-icon-in-a-page-title-on-filamentphp/79031914#79031914
where is the code In my Filament projects I add that trait in app/Filament/Pages/Concerns directory.
